### PR TITLE
New job proposal: 2015-10-26-PhotoFlow-UI_UX-Enhancement

### PR DESCRIPTION
**Pull request by aferrero2707**

### CODE REVIEW

File: jobs/2015-10-26-PhotoFlow-UI_UX-Enhancement [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: jobs
+title: PhotoFlow Image Editor UI/Usability improvement
+role: UI/UX Designer
+organization: PhotoFlow editor
+github: aferrero2707
+contact: photofloweditor@gmail.com
+url: http://aferrero2707.github.io/PhotoFlow/, htpp://http://photoflowblog.blogspot.com/
```
Comment (jancborchardt) [inline jobs/2015-10-26-PhotoFlow-UI_UX-Enhancement]: This link seems to be wrong with the »htpp://« ;)

**Comment by aferrero2707** [comment]
Thanks! I've corrected the blog URL.

**Comment by gillisig** [comment]
The url that gets generated is broken. I think its because the file is missing the ".md" ending. Could you fix that and see if that fixes the url?

**Comment by ei8fdb** [comment]
Should a contact e-mail address/person be added to the posting?

------------
Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by aferrero2707** [comment]
"Should a contact e-mail address/person be added to the posting?"

Is this question addressed to me?

**Comment by ei8fdb** [comment]
I don’t know! It was more a general comment. If someone was interested, how would they get in touch with the person who posted the job.

That was all really :)

------------
Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by aferrero2707** [comment]
Ah, I see... anyhow, the contact email address for the project is given in
the job description. I suppose that this should be enough, right?
Il 27/ott/2015 22:28, "Bernard" <notifications@github.com> ha scritto:

> I don’t know! It was more a general comment. If someone was interested,
> how would they get in touch with the person who posted the job.
>
> That was all really :)
>
> ------------
> Bernard Tyers
> E: ei8fdb@ei8fdb.org | T: @bernardtyers
>
> Talk to me privately. My public key is here: https://keybase.io/ei8fdb
>

**Comment by simonv3** [comment]
@ei8fdb There's a field for contact in the posting. @aferrero2707 had filled that in.

**Comment by ei8fdb** [comment]
Sorry but it’s a bit confusing to me tbh. I hope this will explain what I mean -

On:

http://opensourcedesign.net/jobs/ I see jobs for example:

http://opensourcedesign.net/jobs/jobs/2015-10-26-PhotoFlow-UI_UX-Enhancement.html

There is a link in the Organisation heading

http://opensourcedesign.net/jobs/jobs/2015-10-26-PhotoFlow-UI_UX-Enhancement.html

but only refers to the same page.

If I manually go to the repo and look for the commit:

https://github.com/opensourcedesign/jobs/blob/gh-pages/jobs/2015-10-26-PhotoFlow-UI_UX-Enhancement.md

I see fields with information about email address, URLs, etc.

This is the same with other jobs except some have contact details in the actual description content.

Are they supposed to be viewable on the jobs page? If so they are not there.

If they are not supposed to be there, I think they should be. We can’t expect people to manually get to the repo themselves.

It is possible I am over analysing this!

I hope that makes sense.

**Comment by gillisig** [comment]
@ei8fdb You are correct, the contact is missing from the page. Could you open up an issue for it? Or better yet if you are familiar with jekyll you can submit a pull request to fix it :)

**Comment by simonv3** [comment]
No, you're totally right it is missing! Issue #30 is similar.

**Pull request merged**